products/health-care/appointments/va-online-scheduling/express-care-research/express-care-project-brief.md

```diff
@@ -1,76 +1,71 @@
+## Problem Statement:
+As a Veteran with an urgent medical concern, I need access to a Telehealth appointment with a VA medical provider *today*. 
 
-*******
-# Express Care project brief
+As a Veteran getting medical care from the VA, I don't think there is an urgent Telehealth option available to me.
 
-## Overview: 
-VA devloped Express Care to enable virtual access between Veteran patients and their medical providers ("PACTs"). However, Express Care as a service is not utilized by providers or patients and research needs to be conducted to better understand why and how the VA can improve this service, especially during a time when a Veteran may need medical attention, but be seen at a physical location.
+As a Veteran with a medical concern, I need access to a Telehealth appointment with my VA medical provider *today*, but I am concerned about being seen in-person because of coronavirus.
 
-The problem we are focused on is: understanding when, why, and how a Veteran seeks care that they deem "urgent", "immediate", and "now" through VA digital channels. We specifically want to understand:
- + when do Veterans seek "urgent" "immediate" "same day" care
- + what types of care do Veterans seek "urgent" "immediate" "same day" care / how do they define same day care
- + would Veterans use an online method for receiving same day care / do Veterans know what services are available to them for same day care through VA
+***COVID-19 is creating hot spots throughout the country. These hot spots put a strain on VA medical facilities and staff as they try and field calls and screen patients.***
 
-## Problem Statement:
-Veterans have medical needs that may either be resolved without an in-person appointment and/or want guidance about how seek additional care for a medical concern, yet may not demand a trip to the ER. For various reasons, the Veteran may not be able to go into a physical location for this care. In both cases, however, the Veteran needs to talk to a medical professional *today*.
+As a Veteran who lives in a geographical COVID-19 hot spot, I need to be provided a way get immediate COVID-19 screening and evaluation from my VA doctor/PACT team. 
 
-Currently, "Express Care" is a care option developed by VA for immediate, same day evaluation via VA Video Connect (VA's proprietary telehealth platform). However, Express Care is not utilized by VA providers or patients and we believe the business rules in place prevent this platform from being a preferred option by providers and patients, alike.
+As a VA facility located in a COVID-19 hot spot, I need to be able to screen and evaluate VA patients through a digital medium so that I can limit exposure of VA staff and patients to potential COVID-19 cases and/or alleviate need for in-person evaluation this is creating operational bottlenecks. I want to do this by PACT.
```

Comment:
I think the nurse calling them back through this model may not necessarily be part of their PACT. ANy follow-up appointments would be.